control/tests/statefbk_test.py

```diff
@@ -363,12 +409,21 @@ def test_care(self, matarrayin):
         E = matarrayin(np.identity(2))
         X, L, G = care(A, B, Q, R, S, E, stabilizing=True)
         assert np.all(np.real(L) < 0)
+        
+    @slycotonly
+    def test_care_antistabilizing(self, matarrayin):
+        """Test anti-stabilizing feedbacks, continuous"""
```

Comment:
How about instead of duplicating code lines, you parameterize `test_care` and `test_dare` with something like

```python
@pytest.mark.parametrize(
    "stabilizing",
    [True,
     pytest.param(False, marks=slycotonly),
    ]
def test_care(self, matarrayin, stabilizing):
    ...
    X, L, G = care(A, B, Q, R, S, E, stabilizing=stabilizing)
    sgn = {True: -1,  False: 1}[stabilizing]
    assert np.all((sgn * np.real(L)) > 0)
```

?

Reply:
thanks for the suggestion & code sketch. done